src/Framework/Framework/Storage/FileSystemReturnedFileStorage.cs

```diff
@@ -55,6 +55,12 @@ public FileSystemReturnedFileStorage(string directory, TimeSpan autoDeleteInterv
                 throw new ArgumentNullException(nameof(directory));
             }
 
+            // Check if the directory is a root directory
```

Comment:
I just found out that the check for the root path is not necessary. Can you please remove it?

Reply:
Can you do it in the other service as well? We don't need it there either.